include/dem/dem.h

```diff
@@ -224,8 +224,9 @@ class DEMSolver
    * @param dem_parameters DEM parameters
    * @return A pointer to the particle-particle contact force object
    */
-  std::shared_ptr<PPContactForce<dim>>
-  set_pp_contact_force(const DEMSolverParameters<dim> &dem_parameters);
+  std::shared_ptr<ParticleParticleContactForce<dim>>
+  set_particle_particle_contact_force(
+    const DEMSolverParameters<dim> &dem_parameters);
```

Comment:
Can this be moved outside of the DEM code so that it can be shared with the CFD-DEM coupling.

Reply:
I'll do this in another PR.